portfolio/src/main/webapp/index.html

```diff
@@ -4,23 +4,32 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Comfortaa&display=swap" rel="stylesheet">
+    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
     <script src="script.js"></script>
   </head>
   <body>
+    <div class="home-parallax">
```

Comment:
Could you fix indentation in this div?
Also there are a couple more places in html where indentation is off.